frontend/tests/ui/conftest.py

```diff
@@ -196,6 +197,21 @@ def typeahead_helper(self, parent_xpath, value, select_suggestion=True):
                  '//%s//div[@data-value = "%s"]' %
                  (parent_xpath, value)))).click()
 
+    def typeahead_helper_v2(self, parent_selector, value):
+        '''Helper to interact with Typeahead elements.'''
+
+        tt_input = self.wait.until(
+            EC.visibility_of_element_located(
+                (By.CSS,
```

Comment:
Acá debería ser
```suggestion
                (By.CSS_SELECTOR,
```
, no?

Reply:
Sí! Acabo de corregirlo :D